src/components/TopBar/HamburgerMenu.jsx

```diff
@@ -47,19 +47,19 @@ export default function HamburgerMenu({
       </a>
       <div className="top-bar__menu-item top-bar__menu-item_icons">
         <a
-          className="u__icon"
+          className="u__icon top-bar__menu-item_icon"
```

Comment:
@alexpelan in BEM-speak, this means “the icon variant of the menu item element of the top bar block”, which I don’t think is what we mean (each `<a>` is not itself a menu item). I would probably call this `top-bar__menu-item-icon` (“the menu item icon element of the top bar block”)